### PR TITLE
Fix tutorials test-features CI

### PR DESCRIPTION
**Pull request by 45930**
Fixes a CI failure on `main` from my last commit

### CODE REVIEW

**Comment by 45930** [comment]
@boray - Is there a reason not to run the CI on PRs as well as on push to main?

**Comment by boray** [comment]
> well

yes, to save Github Actions cost. But, you can manually run from Actions tab.

**Comment by boray** [comment]
> @boray - Is there a reason not to run the CI on PRs as well as on push to main?

Is there a reason not to run the CI on PRs as well as on push to main?

**Comment by boray** [comment]
In the context of this PR, you cannot run this version of the workflow in Github Actions. But, you can still test it locally with `act`.

File: .github/workflows/test-tutorials.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+  pull_request:
```
Comment (boray) [inline .github/workflows/test-tutorials.yml:6]: Most of the PRs are not effecting examples so we shouldn't run this workflow on every PR.

File: .github/workflows/test-tutorials.yml [hunk 2]
```diff
@@ -51,7 +52,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [18, 20, 22]
+        node-version: [20, 22]
```
Comment (boray) [inline .github/workflows/test-tutorials.yml:55]: Why not tested on node v18?

Comment (45930) [reply .github/workflows/test-tutorials.yml]: I thought node test runner was introduced in 20, but I guess it will work in 18 as well.  Let's find out.

**Review by boray — APPROVED** [review]
LGTM!

**Pull request merged**